phymem_netbsd.go

```diff
@@ -0,0 +1,41 @@
+package phymem
+
+/*
+#include <stdio.h>
+#include <sys/param.h>
+#include <sys/sysctl.h>
+#include <unistd.h>
+
+static int
```

Comment:
Is size of `int` 64bits on NetBSD always?
If it was 32bits on NetBSD 64bits env., this won't work for over 4GB RSS.

Please check it.

Reply:
I will check your suggestion, however I have a machine trouble.
I will submit revised version later.